src/client/utils/mediaQueries.ts

```diff
@@ -1,11 +1,13 @@
 export const BREAKPOINTS = {
   largeScreen: 1000,
+  mediumLargeScreen: 840,
   mediumScreen: 800,
   mediumSmallScreen: 450,
   smallScreen: 375,
 }
 
 export const LARGE_SCREEN_MEDIA_QUERY = `@media screen and (min-width: ${BREAKPOINTS.largeScreen}px)`
+export const MEDIUM_LARGE_SCREEN_MEDIA_QUERY = `@media screen and (min-width: ${BREAKPOINTS.mediumLargeScreen}px)`
```

Comment:
Do we really need a new breakpoint? 🤔

Reply:
Zak specifically asked for this one : 'Breakpoint from Desktop header to Mobile header should happen sooner (at ~<840px, currently at <800px)' but yeah we can challenge it I guess.

Reply:
it does prevent the arrow overlap, I could add it in this file only I guess